javascript/sdk/lib/types.ts

```diff
@@ -378,6 +394,11 @@ export interface AttachPartnerParams {
   secret?: string;
 }
 
+export interface AttachedPartner {
```

Comment:
I'm a little surprised this didn't already exist? This has been the response for a little while now

Reply:
probably was, the UI didn't use it so the type wasn't update when it was changed.